doc/api/tty.md

```diff
@@ -2,13 +2,19 @@
 
     Stability: 2 - Stable
 
-The `tty` module houses the `tty.ReadStream` and `tty.WriteStream` classes. In
-most cases, you will not need to use this module directly.
+The `tty` modules provides the `tty.ReadStream` and `tty.WriteStream` classes.
+In most cases, it will not be necessary to use this module directly. However,
```

Comment:
In that case, this module really ought to be moved to `lib/internal` with the external API deprecated. The public properties/events that are exposed should be documented via `process.md`.